### ISSUE TITLE
cannot delete an mp3 file played using pygame

### ISSUE THREAD

**Issue by AfrahAsif** [pygame/pygame#955]
I am trying to play and audio file synthsized by Amzon Polly and want to delete the same .mp3 file after it plays.
I am unable to remove a file using os.remove() using the following code,

`import pygame
from pygame.mixer import *
import os
from pygame import mixer

 with open('output.mp3', 'wb') as f:
            f.write(spoken_text['AudioStream'].read())
            f.close()

        mixer.init()
        mixer.music.load('output.mp3')
        mixer.music.play()
        pygame.quit()
        pygame.mixer.init()
        while pygame.mixer.music.get_busy(): 
            pygame.time.Clock().tick(10)

        os.remove('output.mp3')`

The following error is shown:

`PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: output3.mp3`

**Comment by dlon** [comment]
Thanks. As far as I know, the only way to free loaded music at the moment is to load a different track.

**Comment by AfrahAsif** [comment]
Could you @dlon please tell me where should I insert mixer.quit()...Because i have tried that and still same error...so just to know if I am doing it correct or not.

**Comment by dlon** [comment]
This should work, but if you run it more than once within the same program, I think it will only work the first time (due to a bug):
```
    with open('output3.mp3', 'wb') as f:
        f.write(spoken_text['AudioStream'].read())

    mixer.init()
    mixer.music.load('output3.mp3')
    mixer.music.play()
    while pygame.mixer.music.get_busy(): 
        pygame.time.Clock().tick(10)

    mixer.quit()
    os.remove('output3.mp3')
```

**Comment by AfrahAsif** [comment]
Yes. Sadly that is what is happening and what I wanted is to run it multiple time. 
Thanks

**Comment by dlon** [comment]
If you need a quick hack solution, you could load an empty wav/mp3 file before deleting the other one
